lib/misc_functions.rb

```diff
@@ -0,0 +1,75 @@
+# lib/misc_functions.rb
+# Generic implementations of various functions/algorithms that are not crew-specific.
+require 'matrix'
+
+class MiscFunctions
```

Comment:
How about `ConversionFunctions`?  Then the file could be conversion_functions.rb.  Misc is a bit too generic and the name should represent the general purpose of the functions.

Reply:
i agree misc is generic, but these are just a collection of generic functions, thats kinda the point-- only a few of them convert things, so i don't think thats an appropriate name overall.